week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -0,0 +1,54 @@
+/*
+    Given the same "house" object again
+
+    Write the code for the functions as per the description above them
+*/
+
+var house = {
+    address: '1 Kinning Park',
+    price: 180000,
+    currentOwner: {
+      firstName: 'Margaret',
+      lastName: 'Conway',
+      email: "margaret@fake-emails.com"
+    }
+};
+
+var house = {
+    address: '50 Park Avenue',
+    price: 195000,
+    currentOwner: {
+      firstName: 'Marie',
+      lastName: 'McDonald',
+      email: "marie.m@real-emails.com"
+    }
+};
+
+/*
+  DO NOT EDIT ANYTHING ABOVE THIS LINE
+
+  WRITE YOUR CODE BELOW
+*/
+
+// returns the full name (first name + last name) of the owner of the house
+function getOwnerFullName(house) {
+
+}
+
+// returns an array of the owners' email addresses of the two houses
+function getEmailAddresses(house1, house2) {
+
+}
+
+// returns the address for the cheapest house out of the two
+function getCheapestAddress(house1, house2) {

```

Comment:
I wonder if it's confusing having global variables `house1` and `house2`, and the same names as arguments in the functions. Perhaps we could use `margaretHouse` and `marieHouse` for the variable names?

Reply:
Yes! I called them "kinningParkHouse" and "parkAvenueHouse"